src/main/features/claim/routes/pay-by-account.ts

```diff
@@ -8,10 +9,52 @@ import ErrorHandling from 'common/errorHandling'
 import { FeeAccount } from 'forms/models/feeAccount'
 import FeesClient from 'fees/feesClient'
 import ClaimStoreClient from 'claims/claimStoreClient'
-import Claim from 'claims/models/claim'
-import User from 'idam/user'
 import { Fee } from 'fees/fee'
-import MoneyConverter from 'app/fees/moneyConverter'
+
+const logger = require('@hmcts/nodejs-logging').getLogger('router/pay-by-account')
+
+function logError (id: number, message: string) {
+  logger.error(`${message} (User Id : ${id})`)
+}
+
+async function deleteDraftAndRedirect (res, next, externalId: string) {
+  await ClaimDraftMiddleware.delete(res, next)
+  res.redirect(Paths.claimSubmittedPage.evaluateUri({ externalId: externalId }))
+}
+
+async function saveClaimHandler (res, next) {
+  const externalId = res.locals.user.legalClaimDraft.externalId
+
+  let claimStatus: boolean
+  try {
+    claimStatus = await ClaimStoreClient.retrieveByExternalId(externalId)
+      .then(() => true)
+  } catch (err) {
+    if (err.toString().includes('Claim not found by external id')) {
```

Comment:
should be able to check status code, not relying on the message, (I know we do this in T1 but we hacked it together really quickly)

Reply:
done